testeng/jobs/loadtestDriver.groovy

```diff
@@ -131,7 +207,81 @@ job('loadtest-driver') {
 
     publishers {
         archiveArtifacts {
-            pattern('edx-load-tests/results/log.txt')
+            pattern('edx-load-tests/results/*')
+        }
+    }
+}
+
+job('loadtest-summary') {
+
+    description('Summarize a past load test.')
+
+    /* Abusing the team security feature to give all job control
+     * permissions to all edx employees.
+     */
+    authorization JENKINS_PUBLIC_TEAM_SECURITY.call(['edx'])
+
+    parameters {
+        stringParam('REMOTE_BRANCH', 'master',
+                    'Branch of the edx-load-tests repo to use.')
```

Comment:
removed the LOG parameter because the copyArtifacts plugin knows where to find the log on its own.